systemd/systemd.go

```diff
@@ -129,6 +131,40 @@ func Available() error {
 	return err
 }
 
+// Version returns systemd version
+func Version() (int, error) {
+	out, err := systemctlCmd("--version")
+	if err != nil {
+		return 0, err
+	}
+
+	// systemd version outpus is two lines - actual version and a list
+	// of features, e.g:
+	// systemd 229
```

Comment:
This should probably menion that the format can also be `systemd 229 (random-stuff)`

Reply:
Done.